requests_toolbelt/_compat.py

```diff
@@ -18,11 +18,13 @@
     from requests.packages.urllib3 import fields
     from requests.packages.urllib3 import filepost
     from requests.packages.urllib3 import poolmanager
+    from requests.packages.urllib3.util import timeout
```

Comment:
Also this fallback will need to be careful. If we do it unconditionally we can still break existing environments without a new enough requests+urllib3. We will need to do:

```py
try:
   from requests.packages.urllib3.util import timeout
except ImportError:
   try:
       from urllib3.util import timeout
   except ImportError:
       timeout = None
```

Alternatively, we can check if the build number is a certain value and then conditionally attempt to import this with a default value of `None`.